website/views.py

```diff
@@ -4332,3 +4334,17 @@ def deletions(request):
         "monitor.html",
         {"form": form, "deletions": Monitor.objects.filter(user=request.user)},
     )
+
+
+def generate_bid_image(request, bid_amount):
```

Comment:
Optional: it seems the response would be the same for 2 identical `bid_amount` requests. It may be beneficial to keep the result in the cache for a while.

Reply:
thats a good idea thanks